components/tor/tor_launcher_factory.h

```diff
@@ -96,7 +101,11 @@ class TorLauncherFactory : public tor::TorControl::Delegate {
 
   scoped_refptr<base::SequencedTaskRunner> file_task_runner_;
 
-  std::unique_ptr<tor::TorControl> control_;
+  std::unique_ptr<tor::TorControl, base::OnTaskRunnerDeleter> control_;
```

Comment:
for future reference would be extremely useful to add:
 * class-level comments to all major classes (TorControl, TorFileWatcher, etc)
 * field comments describing threading peculiarities: on which sequence the object operates, how it is destroyed, etc

Reply:
fixed in aeed04b506e574973c5df12c9781dde63e187342